api/blueprints/drbEdition.py

```diff
@@ -21,12 +21,14 @@ def editionFetch(editionID):
         or current_app.config['READER_VERSION']
 
     edition = dbClient.fetchSingleEdition(editionID)
-
+            
     if edition:
         statusCode = 200
         records = dbClient.fetchRecordsByUUID(edition.dcdw_uuids)
+        editionWorkTitle = edition.work.title
+        editionWorkAuthors = workAuthorInfo(edition)
         responseBody = APIUtils.formatEditionOutput(
-            edition, records=records, showAll=showAll, reader=readerVersion
+            edition, editionWorkTitle, editionWorkAuthors, records=records, showAll=showAll, reader=readerVersion
```

Comment:
Should we move this into the `APIUtils` module next to `formatEditionOutput`? My gut tells me yes so we can keep the blueprint relatively simple, but not sure if that is moving the functionality to place where its less visible.

Reply:
I'll follow your suggestion and once I commit it, we can see how it ultimately looks.